### PR TITLE
add resource limits to setup job pod

### PR DESCRIPTION
**Pull request by vsinghal13**
###### Description

Solves #740 

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -79,6 +79,13 @@ sumologic:
         helm.sh/hook-weight: "2"
         helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
     job:
+      resources:
```
Comment (frankreno) [inline deploy/helm/sumologic/values.yaml:82]: Have we tested that these are reasonable limits? The setup job really doesn't do a lot, so these seem like they might be high?

Comment (pmalek-sumo) [reply deploy/helm/sumologic/values.yaml:82]: I've tried to look at setup job's utilization and I've noticed values of used memory around 90-120Mi.

Comment (vsinghal13) [reply deploy/helm/sumologic/values.yaml:82]: Thanks @pmalek-sumo for verifying that. I think then the request limit of `128 Mi` is apt.

**Review by pmalek-sumo — APPROVED** [review]

**Review by perk-sumo — APPROVED** [review]
👍

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**